war/src/main/webapp/css/login.theme.css

```diff
@@ -0,0 +1,53 @@
+/*
+IE11 prevents of the usage of variable,
+but to show the color theme we left it here
+ */
+:root {
+    --main-bg-color-focus: #2171ce;
+    --main-bg-color: #4A90E2;
+    --danger-color: #C4000A;
+}
+html {
+    color: #4A4A4A;
+}
+.logo {
+    background-image: url('../images/jenkins.svg');
```

Comment:
does IE support this as a background?

Reply:
![screenshot_2018-04-03_15-41-30](https://user-images.githubusercontent.com/596701/38252876-8293a078-3755-11e8-83ba-466d0f105853.png)